README.md

```diff
@@ -50,14 +50,26 @@ This will automatically load environment variables into your pipenv environment.
     <img src="/README/server_success.png" align="middle" height="70" >
 -   Your Joplin instance will be accessible at http://localhost:8000/admin with the credentials user: `admin@austintexas.io`, pw: `x`
 
-**Run with data**
+**Run with prod data**
 
 ```
 LOAD_DATA="on" ./scripts/serve-local.sh
```

Comment:
I was thinking about this similar lines; maybe eventually we could have a command line param to specify a dump from any environment.

Reply:
That would be cool. But's it would be more complicated than just a command line param. We'd need to build docker images for each branch to run that build's specific migrations. There's a lot going on in migration_test.sh to get to the point where we have a reliably functional datadump. If we get to a point where we frequently need to source data from PR builds we can do it, but for now I'm not sure it'd be worth the dev effort.